backend/todo-list/src/main/java/com/example/todolist/domain/Card.java

```diff
@@ -0,0 +1,98 @@
+package com.example.todolist.domain;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class Card {
+    @Id
```

Comment:
여기엔 한 줄 비워놓으면 좋을 것 같네요.

```suggestion
public class Card {
    
    @Id

Reply:
수정한 뒤 푸쉬하였습니다! 자잘하게 컨벤션(?) 지키지 못한 부분이 많네요